docs/input/docs/localization-howto.md

```diff
@@ -0,0 +1,35 @@
+Description: Localization In ChocolateyGUI
+---
+
+## Basics
+ChocolateyGUI uses `.resx` files for all its strings. 
```

Comment:
This could probably be elaborated a little more. Maybe something like

> To enable using localized strings in the ChocolateyGUI UI, the UI loads all it's from a set of resource files call `resx` files. This allow defining language and culture specific strings and resources, while have english fallbacks when they aren't present. This allows to ChocolateyGUI to, in general, be localization neutral.